tabular/income/model_urls.csv

```diff
@@ -0,0 +1,4 @@
+https://www.dropbox.com/s/em7b6kt4hm7oo8y/model.catb?dl=1,model/model.catb
```

Comment:
oh i see where you generated the models/model/ directory. Since these files are already dumped to a `models` directory, you probably don't need to add a `model` prefix? 

also since the latest PR landed the configs, we may have to edit the config paths as well (we can do it after these PR's land)